conf/cut_release.sh

```diff
@@ -187,6 +192,20 @@ check_branch_dirty() {
     fi
 }
 
+check_npm_registry() {
+    if [[ ! $(npm config get registry) =~ (${YARN_PUBLIC_REGISTRY_REGEX}|${NPM_PUBLIC_REGISTRY_REGEX}) ]] ; then
+        printf "${red}Not pointing at the right npm registry! Make sure .npmrc points to ${NPM_PUBLIC_REGISTRY}${end}"
+        return 1
+    fi
+}
+
+check_npm_login() {
+    if [[ ! $(npm whoami --registry ${NPM_PUBLIC_REGISTRY} 2>/dev/null) ]] ; then
+        printf "${red}Not logged into npm! Try running npm login${end}"
```

Comment:
Can we prompt to login if we reach this step? And read the return value to fail the release script if login doesn't succeed

Reply:
From what I recall, making scripts interactive like this can be kind of a pain and error prone, but let me look into it, but if not, at least the corrective action is included in the error message

Reply:
Should not add prompts to the script. If we decide to move it to Jenkins, then prompts will cause props